src/controllers/machine/index.ts

```diff
@@ -67,18 +67,16 @@ export default class MachineController {
       return response.badrequest({ errors: { name: 'Name is required' } });
```

Comment:
still verifiying if name is on the patch body, which should not be mandatory. remove verification